dspace/modules/journal-submit/src/main/java/org/datadryad/submission/DryadEmailSubmission.java

```diff
@@ -161,6 +149,24 @@ protected void doPost(HttpServletRequest aRequest,
         }
     }
 
+    private void retrieveMail () {
+        LOGGER.info ("retrieving mail with label '" + ConfigurationManager.getProperty("submit.journal.email.label") + "'");
+        try {
+            ArrayList<MimeMessage> messages = DryadGmailService.processJournalEmails();
+            if (messages != null) {
+                LOGGER.info("retrieved " + messages.size() + " messages");
+                for (MimeMessage message : messages) {
+                    try {
+                        processMimeMessage(message);
+                    } catch (Exception details) {
+                        LOGGER.info("Exception thrown while processing MIME message: " + details.getMessage() + ", " + details.getClass().getName());
+                    }
+                }
+            }
+        } catch (IOException e) {
+            LOGGER.info("Exception thrown: " + e.getMessage() + ", " + e.getClass().getName());
```

Comment:
For lines like these, just pass back the whole exception so you can get the stack trace. Like:

LOGGER.info("problem retrieving mail with label", e);